libs/common/src/helpers/get-ukef-deal-id.test.ts

```diff
@@ -0,0 +1,39 @@
+import { BssEwcsDeal, GefDeal } from '../types';
+import { getUkefDealId } from './get-ukef-deal-id';
+import { DEAL_TYPE } from '../constants';
+
+describe('getUkefDealId', () => {
+  it('returns ukefDealId on a GEF deal', () => {
+    // Arrange
+    const ukefDealId = 'ukefDealId';
+
+    const deal = {
```

Comment:
Not for this PR, but my PR starts to look at pulling out these objects from test files into `aBla` functions...